lib/vmpooler.rb

```diff
@@ -79,6 +87,14 @@ def self.new_graphite(server)
     end
   end
 
+  def self.new_statsd(server, port)
+    if server.nil? or server.empty? or server.length == 0
```

Comment:
Prefer `||` to `or`; higher binding precedence when the `if` statement is resolved. I don't think there's a need for the `server.length == 0` either -- it's identical to `server.empty?`:

```ruby
>> server = nil
=> nil
>> server.nil?
=> true
>> server.empty?
NoMethodError: undefined method `empty?' for nil:NilClass [(ripl):3:in `<main>']
>> server.length == 0
NoMethodError: undefined method `length' for nil:NilClass [(ripl):4:in `<main>']
>> server = String.new
=> ""
>> server.nil?
=> false
>> server.empty?
=> true
>> server.length == 0
=> true
```
But the real question is "When would this possibly be `nil` or an empty string?" I recommend skimming Jesse Ezell's "[IsNullOrEmpty](http://weblogs.asp.net/jezell/5280)" for more detail about why checking for `nil`or `#empty?` on an argument could be considered a code smell.